modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/baseline/autoadjust/BaselineTopologyUpdater.java

```diff
@@ -152,6 +150,9 @@ public BaselineAutoAdjustStatus getStatus() {
             if (lastBaselineData.isAdjusted())
                 return BaselineAutoAdjustStatus.notScheduled();
 
+            if (baselineAutoAdjustScheduler.isExecutionExpired(lastBaselineData))
```

Comment:
Do you add separate 'if' consciously or by mistake? ('if' which is above has the same return value)